style.css

```diff
@@ -534,3 +534,150 @@ h6 {
   height: 72px;
   color: #dfe1e6;
 }
+
+.contact-form {
+  background-image: url("./images/contact\ form\ shapes-mobile.png");
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-position: 150px;
+  border-top-left-radius: 90px;
+  overflow-x: hidden;
+  position: absolute;
+  width: 375px;
+  height: 779px;
+  left: 0;
+  top: 4652px;
+}
+
+.last-page {
+  width: 327px;
+  height: 137px;
+  left: 24px;
+  top: 4802px;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  padding: 10px;
+  margin-bottom: 50px;
+
+  /* or 140% */
+  align-items: center;
+  text-align: center;
+  color: #ebebff;
+}
+
+.contact {
+  display: flex;
+  flex-direction: row;
+  width: 801px;
+  height: 56px;
+  left: 50px;
+  top: 4734px;
+  margin-top: 100px;
+  padding-left: 40px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+
+  /* or 130% */
+  align-items: center;
+  text-align: center;
+
+  /* Neutrals /N0 */
+  color: #fff;
+}
+
+.first-type {
+  width: 327px;
+  height: 48px;
+  top: 4991px;
+  left: 24px;
+  padding-left: 10px;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 20px;
+  align-items: center;
+  color: #172b4d;
+}
+
+.second-type {
+  width: 327px;
+  height: 48px;
+  top: 5091px;
+  left: 24px;
+  padding-left: 10px;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 20px;
+  align-items: center;
+  color: #172b4d;
+}
+
+textarea {
```

Comment:
[OPTIONAL]

- [ ] I suggest using CSS classes more preferable than CSS tags because it might be trivial now but once your project becomes bigger & more complex having classes will help in making it easier to add proper styles.

- It's also a good habit to develop :nerd_face:.